config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  namespace 'api' do
+    namespace 'v1' do
+      resources :books
+      get 'api/v1/books/show', to: 'books#show', controller: 'books'
+      get 'api/v1/books/index', to: 'books#index', controller: 'books'
```

Comment:
Prefer use of [resources](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router)

Reply:
solved